cores/esp32/Arduino.h

```diff
@@ -222,6 +222,15 @@ size_t getArduinoLoopTaskStackSize(void);
     return sz;                           \
   }
 
+#define ESP32_USB_MIDI_DEFAULT_NAME "TinyUSB MIDI"
```

Comment:
```suggestion
/**
* @brief Get the current device name
* @return The device name in order of precedence:
* 1. Name set via constructor (if any)
* 2. Name set via SET_USB_MIDI_DEVICE_NAME() macro (if defined)
* 3. Default name "TinyUSB MIDI"
* If device name is set as "", it will be ignored
*/
#define ESP32_USB_MIDI_DEFAULT_NAME "TinyUSB MIDI"
```